internal/pipe/sign/sign_binary_test.go

```diff
@@ -113,9 +115,9 @@ func TestBinarySign(t *testing.T) {
 		require.NoError(tb, pipe.Default(ctx))
 
 		for i := range ctx.Config.BinarySigns {
-			ctx.Config.BinarySigns[i].Args = append(
-				[]string{"--homedir", keyring},
-				ctx.Config.BinarySigns[i].Args...,
+			ctx.Config.BinarySigns[i].Env = append(
+				ctx.Config.BinarySigns[i].Env,
+				"GNUPGHOME="+keyring,
```

Comment:
Setting the GPG home dir as an env variable allows other commands to be used in the test.